common/src/main/java/org/wso2/testgrid/common/config/ConfigurationContext.java

```diff
@@ -267,7 +267,12 @@ public static boolean isDeploymentTinkererEnabled() {
         /**
          * Repeateble json string section of all logs
          */
-        REPEATABLE_ALL_LOGS_JSON("REPEATABLE_ALL_LOGS_JSON");
+        REPEATABLE_ALL_LOGS_JSON("REPEATABLE_ALL_LOGS_JSON"),
+
+        /**
+         * gke service account key file location
+         */
+        GKE_ACCESS_KEY("accessKeyFileLocation");
```

Comment:
This is a duplicate of https://github.com/wso2/testgrid/pull/1210/files#diff-2e8429e596dcb06f4b752d9a93d37f9fR103

Btw, we need to change actual key string as well.